src/component/tooltip/TooltipHTMLContent.ts

```diff
@@ -111,6 +114,8 @@ function assembleTransform(x: number, y: number, toString?: boolean) {
     // If using float on style, the final width of the dom might
     // keep changing slightly while mouse move. So `toFixed(0)` them.
     const x0 = x.toFixed(0) + 'px';
+    // To keep the tooltip slightly above
+    y -= 5;
```

Comment:
What if tooltip is on the bottom/left/right?

Reply:
Ahh I didn't consider it. Let me rework on it. Thanks...

Reply:
Fixed automatically when I adjusted the calculations for the comment below. So removed it.